igcollect/switch.py

```diff
@@ -328,6 +355,23 @@ def ports_stats(prefix, snmp, ports, model):
                 print(template.format(port_name, counter, data))
 
 
+def dom_stats(prefix, snmp, ports, model):
+    """
+    Print graphite-compatible Digital Optical Monitoring stats for each port
+    of the switch
+    """
+
+    for metric, oid in DOM_OIDS[model].items():
+        # SNMP is slow, so we want need to get current time for each counter.
+        template = prefix + '.ports.{}.{} {} ' + str(int(time()))
+        table = get_snmp_table(snmp, oid)
+        for port_idx, port_name in ports.items():
+            if port_idx in table:
+                data = table[port_idx]
+                if data and data != '':
```

Comment:
i like to keep indentations as few as possible. you could negate the ifs and `continue` early if you like

Reply:
I adjusted it like Freddy's suggestion.